fakehydro:
  objects:
    octagon:
      odom_topic: '/coffin_1/pose_gt'
    torpedo:
      odom_topic: '/vampire_stake_1/pose_gt'
    dropper:
      odom_topic: '/dropper_garlic_1/pose_gt'

  meas_freq: 2
  starting_object: dropper # options: 'random', any member of above objecs list
  